src/rgw/rgw_cache.cc

```diff
@@ -104,7 +101,7 @@ int ObjectCache::get(const string& name, ObjectCacheInfo& info, uint32_t mask, r
 bool ObjectCache::chain_cache_entry(std::initializer_list<rgw_cache_entry_info*> cache_info_entries,
 				    RGWChainedCache::Entry *chained_entry)
 {
-  RWLock::WLocker l(lock);
```

Comment:
so, this commit is a revert of a previous one???

Reply:
apologies for the regresion, RWLock::* replaced later on with std::unique_lock